generate_engagement_analysis_files.py

```diff
@@ -103,3 +103,60 @@
 
         for row in opted_in_uids_per_show.items():
             writer.writerow(row)
+
+    log.info(f'Computing repeat and new participation per show ...')
+    # Computes the number of new and repeat consented individuals who participated in each radio show.
+    # Repeat participants are consented individuals who participated in previous shows prior to the target show.
+    # New participants are consented individuals who participated in target show but din't participate in previous shows.
+    repeat_new_participation_map = OrderedDict() # of rqa_raw_field to participation metrics.
+    for rqa_raw_field in rqa_raw_fields:
+        target_radio_show = rqa_raw_field # radio show in which we are calculating participation metrics for.
```

Comment:
Did you get anywhere with friendly names for these (so the exported csv can have e.g. "S01E01" instead of "rqa_s01e01_raw")?

Reply:
will update in a separate pr . thanks

Reply:
Thanks will update in the different pr.